runtime/tools/python/ttrt/common/run.py

```diff
@@ -150,7 +152,7 @@ def initialize_api():
         Run.register_arg(
             name="--golden",
             type=bool,
-            default=True,
+            default=False,
```

Comment:
Intentional? Seems unrelated to this change

Reply:
fixed